integrations/philip-click-up/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,48 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind : team
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"clickupTeam"'
+          properties:
+            url:  \"https://app.clickup.com/\" + .id+ \"/home\"
+            members: "[.members[].user]"
+  - kind : project
+    selector:
+      query: "true" 
+    port:
+      entity:
+        mappings:
+          identifier: .key
+          title: .name
+          blueprint: '"clickupProject"'
+          properties:
+            url: \"https://app.clickup.com/\"+ .folder.id + /"/v/l/\"+ ,id
+            startDate: .start_date
+            endDate: .end_date
+            totalIssues: .task_count
```

Comment:
This will fail due to data type mismatch

Reply:
Thank you, correction ennacted.